src/Hl7.FhirPath/FhirPath/Expressions/CallSignature.cs

```diff
@@ -27,11 +27,11 @@ public CallSignature(string name, Type returnType, params Type[] argTypes )
             ReturnType = returnType;
         }
 
-        public bool Matches(string functionName, IEnumerable<Type> argumentTypes)
-        {
-            return functionName == Name && argumentTypes.Count() == ArgumentTypes.Count() &&
-                   argumentTypes.Zip(ArgumentTypes, (call, sig) => Typecasts.CanCastTo(call,sig)).All(r => r == true);
-        }
+        //public bool Matches(string functionName, IEnumerable<Type> argumentTypes)
```

Comment:
Should we not remove this, if we don't use it

Reply:
We'll need it in the future, when we re-introduce static compilation.